### PR TITLE
Add internal metrics monitoring of the collector

### PR DESCRIPTION
**Pull request by atoulme**

### CODE REVIEW

File: examples/otel-logs-splunk/README.md [hunk 1]
```diff
@@ -14,3 +14,35 @@ $> docker-compose up
 Splunk will become available on port 18000. You can login on [http://localhost:18000](http://localhost:18000) with `admin` and `changeme`.
 
 Once logged in, visit the [search application](http://localhost:18000/en-US/app/search) to see the logs collected by Splunk.
+
+If you would like to see the metrics exposed by the OpenTelemetry Collector, 
+you can visit [http://localhost:8888](http://localhost:8888) to see the metrics emitted by the collector.
+
```
Comment (aurbiztondo-splunk) [inline examples/otel-logs-splunk/README.md:20]: This sentence reads redundant - I'd go for "Visit [http://localhost:8888](http://localhost:8888) to see the metrics emitted by the OpenTelemetry Collector."

**Review by aurbiztondo-splunk — APPROVED** [review]

File: examples/otel-logs-splunk/docker-compose.yml [hunk 2]
```diff
@@ -37,4 +37,15 @@ services:
     depends_on:
       - splunk
     ports:
-      - 18088:8088
\ No newline at end of file
+      - "18088:8088"
+      - "8888:8888"
+  # Prometheus
+  prometheus:
```
Comment (jvoravong) [inline examples/otel-logs-splunk/docker-compose.yml:43]: I'd recommend keeping the otel-logs-splunk example as small as possible, you could create a new example (named something like otel-logs-internal-monitoring) that includes your changes.

**Review by jvoravong — APPROVED** [review]

File: examples/otel-logs-splunk/README.md (hunk 1 above)
Comment (atoulme) [reply examples/otel-logs-splunk/README.md:19]: Addressed, thanks!

File: examples/otel-logs-splunk/docker-compose.yml (hunk 2 above)
Comment (atoulme) [reply examples/otel-logs-splunk/docker-compose.yml:43]: I was on the fence on that. Will do.

**Pull request merged**